integreat_cms/cms/models/contact/contact.py

```diff
@@ -61,7 +63,27 @@ def __str__(self) -> str:
 
         :return: A readable string representation of the contact
         """
-        return f"{self.title} {self.name}"
+        location_name = str(self.location)
+        additional_attribute = self.get_additional_attribute()
+        return _("%s %s") % (location_name, additional_attribute)
```

Comment:
(Probably a nit)
I know the django docs use `%` based formatting, but in integreat_cms we have only used formatting with the `.format` method. There are some problems with `%`-based formatting, which are described here https://peps.python.org/pep-3101/
Even though those problems do not apply to the occurrences here, I would still prefer to use the other formatting syntax for consistency with the rest of the codebase

Reply:
Wait, now that I think about it, does `_("%s %s") % (…)` even make sense at all? We're trying to translate a string that has two spots for values to be inserted. But while it is usually needed for when the sentence structure is switched around in another language so the value has to end up in a different place, here the translation will only be different from the source if we happen to come across a language that does not have spaces…?

Reply:
Is the current implementation solving this issue or is it still the same? :7

Reply:
I'm very much unsure whether this should be considered readable code, but this would be the most exact way to implement this that I can think of:
```python
return " ".join(part for part in [location_name, additional_attribute])
```